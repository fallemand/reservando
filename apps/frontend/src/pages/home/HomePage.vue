<template>
  <Page id="home">TEST</Page>
</template>

<script lang="ts" src="./home.ts" />

<style lang="scss">
@import "~@Assets/styles/variables";

@import "./home__small";
@media screen and (min-width: $screen-sm-max) {
  @import "./home__medium";
}
</style>
