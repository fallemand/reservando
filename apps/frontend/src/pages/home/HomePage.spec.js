import { shallowMount } from "@vue/test-utils";
import HomePage from "@/pages/home/HomePage.vue";
jest.mock("./service");

describe("HomePage", () => {
  it("should render with modules", () => {
    const wrapper = shallowMount(HomePage, {
      global: {
        mocks: {
          $store: {
            dispatch: jest.fn(),
          },
        },
      },
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
