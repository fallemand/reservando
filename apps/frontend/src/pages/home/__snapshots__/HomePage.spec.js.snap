// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage should render with modules 1`] = `
<page-stub
  id="home"
/>
`;
